src/hooks/index.ts

```diff
@@ -457,9 +467,16 @@ export const useWalletMetas = <T = Array<WalletMeta>>(options?: UseQueryOptions<
   return query.data
 }
 
-export const useRemoveWallet = (options: UseMutationOptions<void, Error, void>) => {
+export const useRemoveWallet = ({onSuccess, ...options}: UseMutationOptions<void, Error, void> = {}) => {
+  const dispatch = useDispatch()
+
   const mutation = useMutationWithInvalidations({
     mutationFn: () => walletManager.removeCurrentWallet(),
+    onSuccess: (data, variables, context) => {
+      dispatch(clearUTXOs())
```

Comment:
dont add dispatch to these hooks

Reply:
applied here -> https://github.com/Emurgo/yoroi-mobile/pull/2125